resources/function_help/json/lpad

```diff
@@ -1,11 +1,12 @@
 {
   "name": "lpad",
   "type": "function",
-  "description": "Returns a string padded to supplied width using a fill character.",
+  "description": "Returns a string padded at left to supplied width using a fill character. If the supplied width is smaller than string's length, the string is truncated.",
```

Comment:
"padding on the left to the specified width, using a fill character" sounds a bit better